avocado/utils/multipath.py

```diff
@@ -106,6 +106,37 @@ def get_multipath_wwids():
     return wwids
 
 
+def get_multipath_wwid(mpath):
+    """
+    Get the wwid binding for given mpath name
+
+    :return: Multipath wwid
+    :rtype: str
+    """
+    cmd = "multipathd show maps format '%n %w'"
+    wwids = process.run(cmd, ignore_status=True,
+                        sudo=True, shell=True).stdout_text
+    for wwid in wwids.splitlines(): 
```

Comment:
I think that Travis is complaining about a trailing space here. Not sure.

Reply:
Even I checked there, there is not trailing space at all.  Not sure why it is.